### ISSUE TITLE
What is the output_node_name in the model?

### ISSUE THREAD

**Issue by Albertcc** [weinman/cnn_lstm_ctc_ocr#37]
I want to save the model in PB file mode,Then I can call it in Vc++,But I can't find the output_node_name.
How could I found it?

**Comment by weinman** [comment]
The name given is `[logits](https://github.com/weinman/cnn_lstm_ctc_ocr/blob/7e68b64ed6b54a7da7b7c4dd2db80e0de684e3d4/src/model.py#L167)` but if you want the complete "path" to the name you'll have to search for it.

**Comment by Albertcc** [comment]
Also, I think the output is logits. But then the name "rnn/logits" is a subgraph. 
I tried the name "rnn/logits/Relu" as the output_node_name.
Try to copy the tensor "logits" in train as the output
......
Result = tf.identity(logits,name="output")
......
And all the above, I can't get the same right result as restored the model in CPKT mode.

What should I do??? Help me.

**Comment by weinman** [comment]
I cannot provide a Tensorflow how-to. You might try asking on StackOverflow.

**Comment by shodow** [comment]
@Albertcc I have the same problem,have you solved it?

**Comment by Albertcc** [comment]
No.
When I debug the code, I can see the output tensor named "rnn/logits/Relu:0", But still , I can't get the right output.

**Comment by kapsmeh** [comment]
@Albertcc did you find the input and output nodes yet??
